fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/NotificationService.java

```diff
@@ -280,9 +280,9 @@ private void outputGroupNotification(Map.Entry<String, List<Notification>> entry
         Set<String> groupItems = new HashSet<>();
         for (Notification notification : entry.getValue()) {
             if(displayMessageKeys) {
-                logger.warn("Message Key: " + notification.getKey() + "\n" + notification.getNotificationAsString());
+                groupItems.add("Message Key: " + notification.getKey() + "\n" + notification.getNotificationAsString());
```

Comment:
S: I actually think it makes sense to keep the separate `logger.warn` for displaying the keys as it helps quickly locate the key and understand the divisions between the messages more clearly.

Reply:
If we do as suggested here, it may also make sense to suppress the `logger.warn( groupNotification.getNotificationAsString() )` if `displayMessageKeys` is `true`. (Unless/until we support suppressing entire groups by their key.)

Reply:
I think it would kind of subvert the point of printing out notifications in groups if we break them up with separate logging statements. If a user wanted to copy their whole group of Tiltfile notifications for instance, it would contain a bunch of lines start with [WARN] because of the logging output, so they wouldn't be able to just copy the whole thing and put it in their Tiltfile as is

Reply:
I think if you're trying to find keys to suppress messages, you don't care about the format of the messages you're trying to turn off.